backend/inspirehep/search/facets.py

```diff
@@ -81,7 +81,7 @@ def nested_filters(author_recid):
     return {
         "self_affiliations": must_match_all_filter_nested(
             "authors",
-            "authors.affiliations.value",
+            "authors.raw_affiliations.value",
```

Comment:
this is NOT what should happen. You should have asked me about the requirements. `raw_affiliations` is a completely different field and is NOT what should appear in the facet.